README.adoc

```diff
@@ -141,53 +141,53 @@ include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.1.
 //include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
 //----
 
-Next, we will define some basic information that informs MicroShed Testing if and how it should start our applictaion runtime, and what URL path the application will be available at:
+Next, this guide outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
```

Comment:
The original `we will define` phrasing seems to make more sense, but if we can't use the term "we" then the next best thing would be something like "Next, the `PersonServiceIT` outlines"

Reply:
We do want to avoid the "we"/"our" language. But good on "Next, the `PersonServiceIT` class outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:"